packages/contracts-periphery/contracts/foundry-tests/AssetReceiver.t.sol

```diff
@@ -17,6 +17,21 @@ contract AssetReceiver_Initializer is Test {
     TestERC721 testERC721;
     AssetReceiver assetReceiver;
 
+    event ReceivedETH(address indexed from, uint256 amount);
+    event WithdrewETH(address indexed withdrawer, address indexed recipient, uint256 amount);
+    event WithdrewERC20(
+        address indexed withdrawer,
+        address indexed recipient,
+        address indexed asset,
+        uint256 amount
+    );
+    event WithdrewERC721(
+        address indexed withdrawer,
+        address indexed recipient,
+        address indexed asset,
+        uint256 id
+    );
+
     function _setUp() public {
```

Comment:
I think we should use this PLR as an opportunity to start moving away from this pattern.
I.e. if we make the following change, then we can delete the other `setUp` functions (which are just thin wrappers) from the child contracts: 
```suggestion
    function setUp() public {
```

Reply:
I have cleaned this up, good call